aspnetcore/signalr/configuration.md

```diff
@@ -165,17 +165,7 @@ let connection = new signalR.HubConnectionBuilder()
 > [!NOTE]
 > To disable logging entirely, specify `signalR.LogLevel.None` in the `configureLogging` method.
 
-Log levels available to the JavaScript client are listed below. Setting the log level to one of these values enables logging of messages at **or above** that level.
-
-| Level | Description |
```

Comment:
I moved this to the Diagnostics page because it felt like more detailed info belonged there (that's where I put custom loggers as well). Happy to move it back or duplicate it in both if people think it's better.